### PR TITLE
Intro text

### LINKED ISSUE #29: Add Intro to project text to index
## User story
As a person visiting the site, I need to see some explanation of the core mission of act, connect, share so that I know what this is all about and can take part meaningfully.

## Acceptance criteria
- [x] There is some intro text on the index  page
- [x] I can see a `[cCc]` at the start of this text so I know it needs to be approved

## Implementation notes & questions

The text should be put in the Copy module. Later on we will want to pull this from markdown

### PR DESCRIPTION
**Pull request by ivan-kocienski-gfsc**
Fixes #29 

Intro text copy added to language file and main view function

@geeksforsocialchange/developers

### CODE REVIEW

File: src/Theme/PageTemplate.elm [hunk 1]
```diff
@@ -30,6 +30,9 @@ view pageInfo =
         [ HeaderTemplate.view { content = t pageInfo.title }
         , main_ [ css [ mainStyle ] ]
             [ button [ onClick LanguageChangeRequested ] [ text (t ChangeLanguage) ]
+            , p [] [ text (t WelcomeP1) ]
```
Comment (katjam) [inline src/Theme/PageTemplate.elm:33]: @aaaaargZombies @r-ferrier Should we stick a container around these `p`'s?

**Comment by aaaaargZombies** [comment]
I think this is appearing on all pages not just index

File: src/Theme/PageTemplate.elm (hunk 1 above)
Comment (aaaaargZombies) [reply src/Theme/PageTemplate.elm:33]: maybe a container with something like 

```css
.center {
  box-sizing: content-box;
  max-inline-size: 60ch;
  margin-inline: auto;
  display: flex;
  flex-direction: column;
  align-items: center;
}
```

if we're getting into layout now, but maybe beyond scope of ticket

Comment (r-ferrier) [reply src/Theme/PageTemplate.elm:33]: Feels out of scope to me?

Comment (katjam) [reply src/Theme/PageTemplate.elm:33]: I want the foundational layout, markup and accessibility to be addressed as we add stuff here, even though we have no theme as yet. Just the basics like containers so responsiveness is by default.

**Comment by katjam** [comment]
> I think this is appearing on all pages not just index

Oops. Good spot! I'll unapprove.

**Pull request merged**